rust/src/container.rs

```diff
@@ -71,6 +71,11 @@ struct ContainerEncapsulateOpts {
     /// Compare OCI layers of current build with another(imgref)
     #[clap(name = "compare-with-build", long)]
     compare_with_build: Option<String>,
+
+    /// Prevent a change in packing structure by taking a previous build metadata (oci config and
+    /// manifest)
+    #[clap(long, short)]
```

Comment:
Did you explicitly mean to add `short` here?  I think that will make this open show up as `-p` too...which is probably fine, but short options I think should be reserved for *very* frequently typed things.  

This particular command honestly I feel like 99% of the time isn't going to be typed by a human, so we can default to verbose names.